packages/interfaces/src/store.ts

```diff
@@ -73,6 +75,13 @@ export type QueryRequestParams = {
    * @default undefined
    */
   paginationLimit?: number;
+
+  /**
+   * The service node to use for queries. Will fail if this peer is not in the
+   * peer store. No fallback is done. Overrides any other peer selection option.
+   * Only use if you know what you are doing.
+   */
+  peerId?: PeerId;
```

Comment:
I think the same result can be reached with https://github.com/waku-org/js-waku/blob/0dfbcf6b6bd9225dcb0dec540aeb1eb2703c8397/packages/interfaces/src/store.ts#L106

Reply:
No... I want to do one query to one specific peer.

You are pointing to an option that is for "preferred " peers, and set when mounting the store protocol.

2 different logics.

Reply:
I am doing KISS. I only use this API upon a StoreConnect event. Hence, I am not convinced we should add complexity "just in case". But I will review the API doc to ensure it is crystal clear.